nodes/parachain/src/command.rs

```diff
@@ -79,15 +79,15 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, St
 	log::info!("The following runtime was chosen based on the spec id: {}", runtime);
 
 	match (id, runtime) {
-		("dev", _) => Ok(Box::new(chain_spec::peregrine::make_dev_spec()?)),
+		("dev", _) => Ok(Box::new(chain_spec::peregrine::get_chain_spec_dev()?)),
```

Comment:
I guess adding `peregrine` somewhere in the name of the function would make it less confusing.